### PR TITLE
Improve api keys documentation in readme

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,9 +98,9 @@ First, you need to add your MeiliSearch credentials.
 
 The credentials are composed of:
 - The `host`: The url to your running MeiliSearch instance.
-- The `api_key`: The `master` or `private` key as the plugin requires administration permission on MeiliSearch.[More about permissions here](https://docs.meilisearch.com/reference/features/authentication.html).
+- The `api_key`: The `master` key or another `key` with the permission to add documents in MeiliSearch. [More about permissions and api keys here](https://docs.meilisearch.com/learn/advanced/security.html).
```
Comment (curquiza) [inline README.md:101]:
```suggestion
- The `api_key`: The `master` key or another `key` with the permission to add documents in MeiliSearch. [More about permissions and API keys here](https://docs.meilisearch.com/learn/advanced/security.html).
```

File: README.md [hunk 2]
```diff
@@ -98,9 +98,9 @@ First, you need to add your MeiliSearch credentials.
...
 
-⚠️ The `master` or `private` key should never be used to `search` on your front end. For searching, use the `public` key available on [the `key` route](https://docs.meilisearch.com/reference/api/keys.html#get-keys).
+⚠️ Keys with permissions other than `search` should never be used on your front end. For searching, use the `Default Search Key` key available on [the `key` route](https://docs.meilisearch.com/reference/api/keys.html#get-keys) or [create a `custom api key`](https://docs.meilisearch.com/learn/advanced/security.html) with only search rights.
```
Comment (curquiza) [inline README.md:103]:
```suggestion
⚠️ Keys with permissions other than `search` should never be used on your front end. For searching, use the `Default Search Key` key available on [the `key` route](https://docs.meilisearch.com/reference/api/keys.html#get-keys) or [create a custom API key](https://docs.meilisearch.com/learn/advanced/security.html) with only search rights.
```

**Review by curquiza — COMMENTED** [review]
💪

**Review by brunoocasali — APPROVED** [review]
🔥

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**